server/server/src/main/kotlin/org/jetbrains/bsp/bazel/server/diagnostics/CompilerDiagnosticParser.kt

```diff
@@ -9,28 +12,33 @@ object CompilerDiagnosticParser : Parser {
   // server/DiagnosticsServiceTest.kt:12:18: error: type mismatch: inferred type is String but Int was expected
   private val DiagnosticHeader =
     """
-      ^                # start of line
-      ([^:]+)          # file path (1)
-      :(\d+)           # line number (2)
-      (?::(\d+))?      # optional column number (3)
-      :\               # ": " separator
-      ([a-zA-Z\ ]+):\  # level (4)
-      (.*)             # actual error message (5)
+      ^                       # start of line
+      (?<logLevel>\[.*\]\s*)? #optional
+      (?<filePath>[^:]+)      # file path (2)
+      :(?<lineNumber>\d+)     # line number (3)
+      (?::(?<columnNumber>\d+))?      # optional column number (4)
+      (?::\ )?                # ": " separator
+      (?:(?<errorLevel>[a-zA-Z\ ]+):\ )? # optional level (5) could have been at the beginning instead
+      (?<errorMessage>.*)             # actual error message (6)
       $                # end of line
       """.toRegex(RegexOption.COMMENTS)
 
-  fun tryParseOne(output: Output): Diagnostic? =
-    output
+  fun tryParseOne(output: Output): Diagnostic? {
+    LOGGER.info("Checking output:\n${output.peek()}")
```

Comment:
Should we keep this debugging statement?

Reply:
Nope